jlink.go

```diff
@@ -155,12 +155,12 @@ func (j *JLink) buildCustomJRE(layerPath string) error {
 func (j *JLink) validArgs() bool {
 	jlinkArgs := j.Args[:0]
 	var skipNext, modsFound bool
-	for _, a := range j.Args {
+	for _, original := range j.Args {
 		if skipNext {
 			skipNext = false
 			continue
 		}
-		a = strings.ToLower(a)
+		var a = strings.ToLower(original)
```

Comment:
```suggestion
		a := strings.ToLower(original)
```

I think it's preferred to use the short variable declaration where possible. I think you should be able to use it here.